chef_master/source/platforms.rst

```diff
@@ -56,16 +56,16 @@ The following table lists the commercially-supported platforms and versions for
      - ``x86_64``, ``s390x``, ``ppc64le`` (7.x only), ``ppc64`` (7.x only)
      - ``6.x``, ``7.x``
    * - Solaris
-     - ``sparc``, ``x86``
+     - ``sparc``, ``i86pc``
      - ``11.2``, ``11.3``, ``11.4``
    * - SUSE Enterprise Linux Server
      - ``x86_64``, ``s390x``, ``ppc64le``, ``ppc64``
      - ``11 SP4``, ``12 SP1+``
    * - Ubuntu (LTS releases)
-     - ``x86``, ``x86_64``
+     - ``i386``, ``x86_64``
```

Comment:
So... we haven't started building on 16.06 i386... is there a plan to do that? Otherwise this becomes just x86_64. I don't want to block this PR on that other discussion, just a comment.